guardian/concierge-graphql
# allow login to AppSync style backends too

## What does this change?

Adds an option to select the API path, allowing AppSync style URIs (/graphql) as well as conventional ones (/query)

## How to test

- Use the 'GraphQL endpoint' param from the AppSync api "fronts-graphql-test" in the CAPI account
- Use an API key from the configuration of that API
- Select the "[Appsync] api type and it should work. If you don't you'll see an error along the lines "unexpected operation"

## How can we measure success?

Able to talk to AppSync

## Have we considered potential risks?

n/a

## Images

![Screenshot from 2024-05-24 15-11-51](https://github.com/guardian/concierge-graphql/assets/12482441/3547c32f-12a3-4bf3-8509-d826aa67c01d)
